tests/test_cache_parallel_performance.py

```
@@ -0,0 +1,525 @@
+        # Aguarda conclusão
+        for thread in threads:
+            thread.join()
+
+        total_time = time.time() - start_time
+
+        # Valida resultados
+        assert len(results) == num_threads
+        total_items = sum(r["success_count"] for r in results)
+        total_throughput = total_items / total_time
+
+        print(f"\n📊 PERFORMANCE DE ESCRITAS CONCORRENTES:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)